.claude/skills/fix-hypershift-repo-robot-pr/SKILL.md

```
@@ -0,0 +1,275 @@
+
+```bash
+gh pr view "$PR_NUMBER" --json number,title,author,headRefName,baseRefName,body,state,url,commits
+```
+
+**Required validations:**
+- PR must exist and be `OPEN`
+- `author.login` must match one of the trusted bots listed below
+- Working directory must be clean (no uncommitted changes)
+
+**Trusted bot allowlist:**
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)